### PR TITLE
Use "most" instead of "more", in singular for clarity and consistency

### PR DESCRIPTION
**Pull request by fluorine**

### CODE REVIEW

File: book/01-introduction/sections/about-version-control.asc [hunk 1]
```diff
@@ -22,7 +22,7 @@ To deal with this issue, programmers long ago developed local VCSs that had a si
 .Local version control.
 image::images/local.png[Local version control diagram]
 
-One of the more popular VCS tools was a system called RCS, which is still distributed with many computers today.
```
Comment (Frezzle) [inline book/01-introduction/sections/about-version-control.asc]: If using `most` here then `tool` is wrong; `tools` is still correct. Some suggestions:
- `One of the most popular VCS tools was `...
- `The most popular VCS tool was `...

Comment (fluorine) [inline book/01-introduction/sections/about-version-control.asc]: I agree.

File: book/01-introduction/sections/about-version-control.asc [hunk 2]
```diff
@@ -22,7 +22,7 @@ To deal with this issue, programmers long ago developed local VCSs that had a si
...
+One of the most popular VCS tool was a system called RCS, which is still distributed with many computers today.
```
Comment (ben) [inline book/01-introduction/sections/about-version-control.asc]: Unfortunately now we have a disagreement. This should read "One of the most popular VCS *tools* …"

**Comment by parkinson101** [comment]
#1016

**Comment by aollier** [comment]
@ben I propose to make the correction myself and close this pull request. Do you agre?

**Comment by ben** [comment]
Sure, go ahead.

**Pull request closed without merging**